src/parameters/parameters_flow_solver.cpp

```diff
@@ -91,6 +91,10 @@ void FlowSolverParam::declare_parameters(dealii::ParameterHandler &prm)
                           dealii::Patterns::Bool(),
                           "Solve steady-state solution. False by default (i.e. unsteady by default).");
 
+        prm.declare_entry("error_adaptive_time_step", "false",
+                          dealii::Patterns::Bool(),
+                          "Adapt the time step on the fly for unsteady flow simulations. False by default (i.e. constant time step by default).");
```

Comment:
Could you modify this comment and the one for `adaptive_time_step` to describe how the two are different? e.g., your new method would say "adapt according to an estimate of temporal error" (or words to that effect), and the old one would say "adapt according to a CFL condition".